contracts/apps/disputable/DisputableAragonApp.sol

```diff
@@ -145,6 +146,7 @@ contract DisputableAragonApp is IDisputable, AragonApp {
     * @dev Close action in the Agreement
     * @param _actionId Identifier of the action in the context of the Agreement
     */
+   // Note
     function _closeAgreementAction(uint256 _actionId) internal {
```

Comment:
Similarly, this would be `_closeDisputableAction()` or `_finishDisputableAction()`.